operations/mimir-mixin/dashboards/remote-ruler-reads.libsonnet

```diff
@@ -10,6 +10,7 @@ local filename = 'mimir-remote-ruler-reads.json';
     assert std.md5(filename) == 'f103238f7f5ab2f1345ce650cbfbfe2f' : 'UID of the dashboard has changed, please update references to dashboard.';
     ($.dashboard('Remote ruler reads') + { uid: std.md5(filename) })
     .addClusterSelectorTemplates()
+    .addShowNativeLatencyVariable()
```

Comment:
Is this intentionally placed here?

Reply:
yes, the query-frontend panels are common. see diff in the compiled files for remote-ruler-reads.